### PR TITLE
[CMSP-1034] fix typo in tax_query

### PR DESCRIPTION
**Pull request by jazzsequence**
Adds customer-supplied code for tax_queries from https://wordpress.org/support/topic/issue-building-query-when-you-are-parsing-the-tax_query/ props [@offshorealert](https://wordpress.org/support/users/offshorealert/),

### CODE REVIEW

File: includes/class-solrpower-wp-query.php [hunk 1]
```diff
@@ -740,7 +740,7 @@ private function parse_tax_query( $tax_query ) {
 					$multi_query   = array();
 					$multi_query[] = '(' . $field . ':(' . implode( 'OR', $terms ) . '))';
 					if ( $tax_value['include_children'] && is_taxonomy_hierarchical( $tax_value['taxonomy'] ) ) {
-						$multi_query[] = '(parent_' . $field . ':' . implode( 'OR', $terms ) . ')';
+						$multi_query[] = '(parent_' . $field . ':' . implode( 'OR', $terms ) . '))';
```
Comment (pwtyler) [inline includes/class-solrpower-wp-query.php:743]: you're missing the sad emoji in the middle 
```suggestion
						$multi_query[] = '(parent_' . $field . '(:' . implode( 'OR', $terms ) . '))';
```

from https://wordpress.org/support/topic/issue-building-query-when-you-are-parsing-the-tax_query/

**Comment by jazzsequence** [comment]
Behat tests that are failing are on multidev deletion, the actual _tests_ are passing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ## Changelog ##
 
-### 2.5.3-dev ###
+### 2.5.3 (April 24, 2024) ###
```
Comment (pwtyler) [inline CHANGELOG.md:3]: This should happen in release PR, not feature branch.

File: bin/behat-cleanup.sh [hunk 3]
```diff
@@ -20,4 +20,4 @@ set -ex
 ###
 # Delete the environment used for this test run.
 ###
-terminus multidev:delete $SITE_ENV --delete-branch --yes
+terminus multidev:delete $SITE_ENV --delete-branch --yes || true
```
Comment (pwtyler) [inline bin/behat-cleanup.sh:23]: please at least stub a ticket to understand why this is happening if its persistent.

File: readme.txt [hunk 4]
```diff
@@ -235,7 +235,8 @@ Please report security bugs found in the source code of the Solr Power plugin th
 
 == Changelog ==
 
-= 2.5.3-dev =
+= 2.5.3 (April 24, 2024) =
```
Comment (pwtyler) [inline readme.txt:238]: as above this should be release pr but fine if you're shipping this manually.

**Review by pwtyler — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jazzsequence) [reply CHANGELOG.md:3]: I branched off main and then went back and changed things when I realized my mistake. See https://github.com/pantheon-systems/solr-power/pull/622/commits/ee583cfed45bf81aa3f56b5f2bed78918435d058

File: bin/behat-cleanup.sh (hunk 3 above)
Comment (jazzsequence) [reply bin/behat-cleanup.sh:23]: #623

**Pull request merged**